codebuild/linux-clang6-x64-downstream.yml

```diff
@@ -0,0 +1,24 @@
+version: 0.2
+#this buildspec assumes the ubuntu 14.04 trusty image
+phases:
+  install:
+    commands:
+      - wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | sudo apt-key add -
+      - sudo add-apt-repository ppa:ubuntu-toolchain-r/test
+      - sudo apt-add-repository "deb http://apt.llvm.org/trusty/ llvm-toolchain-trusty-6.0 main"
+      - sudo apt-get update -y
+      - sudo apt-get install clang-6.0 cmake3 clang-tidy-6.0 clang-format-6.0 libssl-dev -y -f
+
+  pre_build:
+    commands:
+      - export CC=clang-6.0
+      - export CXX=clang++-6.0
```

Comment:
why is a C++ compiler involved anywhere?

Reply:
copy pasta :)